youtube_dl/extractor/crunchyroll.py

```diff
@@ -440,11 +423,9 @@ def _real_extract(self, url):
         language = self._search_regex(
             r'(?:vilos\.config\.player\.language|LOCALE)\s*=\s*(["\'])(?P<lang>(?:(?!\1).)+)\1',
             webpage, 'language', default=None, group='lang')
-
-        video_title = self._html_search_regex(
-            r'(?s)<h1[^>]*>((?:(?!<h1).)*?<span[^>]+itemprop=["\']title["\'][^>]*>(?:(?!<h1).)+?)</h1>',
-            webpage, 'video_title')
-        video_title = re.sub(r' {2,}', ' ', video_title)
+        video_title = self._html_search_meta('title', webpage) or self._html_search_regex(r'<title>(?P<title>.+?), - Watch on Crunchyroll</title>',
+                                                                                          webpage, 'video_title', group='title')
+        video_title = re.sub(r', - Watch on Crunchyroll', '', video_title)
```

Comment:
This produces invalid results when being called in a non-english speaking country.
At least for me, in germany, titles will have `, - Anschauen auf Crunchyroll` appended (which is the same phrase being cut off here, just in german)
This was not the case before

Reply:
Thank you for pointing this out, I have made some changes to the regex. I have also added another test for a pt-PT link.